lighthouse-core/gather/computed/critical-request-chains.js

```diff
@@ -67,7 +71,8 @@ class CriticalRequestChains extends ComputedArtifact {
         startTime: request.startTime,
         endTime: request.endTime,
         responseReceivedTime: request.responseReceivedTime,
-        transferSize: request.transferSize
+        transferSize: request.transferSize,
+        blocked: request.statusCode === 0
```

Comment:
yup, there's a better way. Here's part of the `networkRequest` object from devtools: https://github.com/ChromeDevTools/devtools-frontend/blob/master/front_end/sdk/NetworkRequest.js#L404-L423